man/Makefile

```diff
@@ -0,0 +1,13 @@
+Q := @
+
+clean:
+	rm -f elixir.1
+	rm -f iex.1
+
+build:
+	$(Q) cp iex.1.in iex.1
+	$(Q) sed -i"" "/{COMMON}/r common" iex.1
```

Comment:
FWIW, I had to add a space between `-i` and `""` because the sed implementation in my mac complained. So it became `sed -i "" "/{COMMON}/r common" iex.1` and so on.